rust/feature-flags/Cargo.toml

```diff
@@ -39,6 +39,8 @@ health = { path = "../common/health" }
 common-metrics = { path = "../common/metrics" }
 tower = { workspace = true }
 derive_builder = "0.20.1"
+petgraph = "0.6.5"
```

Comment:
I use this for building cohort dependency graphs, IMO better than write my own topographical sort.